gatekeeper-3.12.yaml

```diff
@@ -0,0 +1,54 @@
+package:
+  name: gatekeeper-3.12
+  version: 3.12.0
+  epoch: 0
+  description: Gatekeeper - Policy Controller for Kubernetes
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    provides:
+      - gatekeeper=3.12
```

Comment:
I think we had a `3.12`, so this needs to be `3.12.999` (see `consul-1.15` for example)

Reply:
Done